pkg/cmd/roachtest/tests/rust_postgres.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/option"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
+)
+
+func registerRustPostgres(r registry.Registry) {
+	runRustPostgres := func(ctx context.Context, t test.Test, c cluster.Cluster) {
+		if c.IsLocal() {
+			t.Fatal("can not be run in local mode")
+		}
+		node := c.Node(1)
+		t.Status("setting up cockroach")
+		c.Put(ctx, t.Cockroach(), "./cockroach", c.All())
+		c.Start(ctx, t.L(), option.DefaultStartOpts(), install.MakeClusterSettings(), c.All())
```

Comment:
if we need this to be port 5433, we should add an option to https://github.com/cockroachdb/cockroach/blob/5af48f0e85f10124161679408024b29623d78f73/pkg/roachprod/install/cockroach.go#L105 and make it take in 5433 as an arg, and augment `--port` to this port number. alternatively, can we change the PGURL to use 26257 instead?